ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts

```diff
@@ -30,8 +30,8 @@ export class PokeFilterTimeTabComponent implements OnInit {
     setTimeout(() => this.cdr.detectChanges(), 100);
   }
 
-  onSightingsRangeChanged(event) {
-    this.filter.sightingsRange = event.value;
+  onSightingsRangeChanged(value) {
```

Comment:
Is there a reason we input a `event` in `poke-filter-time-tab/poke-filter-time-tab.component.html` and get out a `value`? Could this be named better, or do I just not understand it? ^^